src/systems/triggered_publisher/TriggeredPublisher.cc

```diff
@@ -598,31 +612,73 @@ void TriggeredPublisher::Configure(const Entity &,
   else
   {
     ignerr << "No ouptut specified" << std::endl;
```

Comment:
Since we are now making it so that you can have an output *or* a service (or both), I think this error message should be removed.